src/views/guests/Login.vue

```diff
@@ -1,105 +1,150 @@
 <template>
-  <div class="login row">
-    <div class="col-3 col-md-5 col-sm-4 "></div>
-    <div class="login-in col-6 col-md-2 col-sm-4">
-      <br />
-      <h1 class="login-h1">Login</h1>
-      <input
-        type="text"
-        id="username"
-        name="username"
-        v-model="username"
-        placeholder="Username"
-        require
-      />
-      <br />
-      <input
-        type="password"
-        id="password"
-        name="password"
-        v-model="password"
-        placeholder="Password"
-        require
-      />
-      <br />
+  <div class="login">
+    <div class="login-in">
```

Comment:
`login-in` typo mu yoksa login sayfasının içi demek mi?Gerekli değişikliği yapınız